File: Microservice_Python/microservices.py
from flask import Flask, jsonify

app = Flask(__name__)

@app.route('/')
def get_introduction():
    return "Bienvenido a mi microservicio con Flask + Python 3.9!.."

@app.route('/message1')
def get_message1():
    return "Hola Mundo!"

@app.route('/message2')
def get_message2():
    return "Hola Valle Grande!"

@app.route('/message3')
def get_message3():
    jsonResponse = {
        "message": "Hola desde getMessage3",
        "author": "Pedro Porras Luyo",
        "tags": ["Flask", "Python 3.9", "API"]
    }
    return jsonify(jsonResponse)

if __name__ == '__main__':
    app.run(host='0.0.0.0', port=8080)

File: Microservice_Node/microservices.js
const express = require('express');
const app = express();

const port = 8080;

app.get('/', (req, res) => {
  res.send('Bienvenido a mi microservicio con Node.js!');
});

app.get('/message1', (req, res) => {
  res.send('Hola Mundo!');
});

app.get('/message2', (req, res) => {
  res.send('Hola Valle Grande!');
});

app.get('/message3', (req, res) => {
  const jsonResponse = {
    message: 'Hola desde getMessage3',
    author: 'Pedro Porras Luyo',
    tags: ['Node.js', 'Express', 'API']
  };

  res.json(jsonResponse);
});

app.listen(port, () => {
  console.log(`El microservicio está corriendo en el puerto ${port}`);
});
